pkg/admin/transport/handler_user_export_get.go

```diff
@@ -46,7 +48,7 @@ func (h *UserExportGetHandler) handle(w http.ResponseWriter, r *http.Request) er
 		return err
 	}
 
-	response, err := userexport.NewResponseFromTask(task)
+	response, err := userexport.NewResponseFromTask(task, h.CloudStorage)
```

Comment:
Keep NewResponseFromTask unchanged.

Generate the presigned url here.

Reply:
moved sign url to here